File: java/darts/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double distance = Math.sqrt(Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2));

        return distance <= 1 ? 10 : distance <= 5 ? 5 : distance <= 10 ? 1 : 0;
    }
}

File: javascript/darts/darts.js
export const score = (x, y) => {
    let d = Math.sqrt((x ** 2) + (y ** 2));

    if (d <= 1) {
        return 10;
    }

    if (d <= 5) {
        return 5;
    }

    if (d <= 10) {
        return 1;
    }

    return 0;
};
